app/helpers/container_project_helper/textual_summary.rb

```diff
@@ -16,6 +16,59 @@ def textual_group_smart_management
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_quota
```

Comment:
@simon3z UI changes look good, don't have data to test but screenshot looks fine. @blomquisg @fryguy please review model changes.